src/test/java/com/jcabi/github/RtContentsITCase.java

```diff
@@ -257,6 +259,70 @@ public void getContent() throws Exception {
     }
 
     /**
+     * RtContents can iterate content.
+     * @throws Exception If some problem inside
+     * @todo #863 unignore after Contents#get is implemented for
+     *  directories (#968 and #903)
+     */
+    @Test
+    @Ignore
+    public void iteratesContent() throws Exception {
+        final Repos repos = github().repos();
+        final Repo repo = this.rule.repo(repos);
+        try {
+            final String afile = RandomStringUtils.randomAlphanumeric(Tv.TEN);
+            final String dir = RandomStringUtils.randomAlphanumeric(Tv.TEN);
+            final String bfile = String.format(
+                "%s/%s",
+                dir,
+                RandomStringUtils.randomAlphanumeric(Tv.TEN)
+            );
+            final String message = String.format("testMessage");
+            final Contents contents = repos.get(repo.coordinates()).contents();
+            contents.create(
+                this.jsonObject(
+                    afile,
+                    new String(
+                        Base64.encodeBase64(
+                            String.format(
+                                "content a:%d",
+                                System.currentTimeMillis()
+                            )
```

Comment:
Can't, will be 84 symbols long line even if I move `format()` call to a separate string.